pkg/agent/querier/querier_test.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package querier
+
+import (
+	"net"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	interfacestoretest "github.com/vmware-tanzu/antrea/pkg/agent/interfacestore/testing"
+	openflowtest "github.com/vmware-tanzu/antrea/pkg/agent/openflow/testing"
+	"github.com/vmware-tanzu/antrea/pkg/apis/clusterinformation/v1beta1"
+	binding "github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
+	ovsconfigtest "github.com/vmware-tanzu/antrea/pkg/ovs/ovsconfig/testing"
+	queriertest "github.com/vmware-tanzu/antrea/pkg/querier/testing"
+)
+
+func getIPNet(ip string) *net.IPNet {
+	_, ipNet, _ := net.ParseCIDR(ip)
+	return ipNet
+}
+
+func TestAgentQuerierGetAgentInfo(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	interfaceStore := interfacestoretest.NewMockInterfaceStore(ctrl)
+	interfaceStore.EXPECT().GetContainerInterfaceNum().Return(2).AnyTimes()
+
+	ofClient := openflowtest.NewMockClient(ctrl)
+	ofClient.EXPECT().GetFlowTableStatus().Return([]binding.TableStatus{
+		{
+			ID:        1,
+			FlowCount: 2,
+		},
+	}).AnyTimes()
+	ofClient.EXPECT().IsConnected().Return(true).AnyTimes()
+
+	ovsBridgeClient := ovsconfigtest.NewMockOVSBridgeClient(ctrl)
+	ovsBridgeClient.EXPECT().GetOVSVersion().Return("2.10.0", nil).AnyTimes()
```

Comment:
maybe declare a variable `ovsVersion` for `"2.10.0"`

Reply:
done